### PR TITLE
Splited SignUp page in two blocks

### PR DESCRIPTION
**Pull request by mdamonte88**
Splited SignUp page in two blocks and added the right content with a phone and other icons

signupenglish

signupspanish

signuperrors

### CODE REVIEW

**Comment by bernabe9** [comment]
Please provide some screenshots, so I can compare them with the design

**Comment by bernabe9** [comment]
screen shot 2018-10-30 at 6 59 38 pm

I see some difference with the design:
• Font family & size
• Paddings
• Input width

**Comment by bernabe9** [comment]
Keep only one version of the images. You don't need to keep three versions of them

File: src/components/common/Select.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+export default class Select extends PureComponent {
+  static propTypes = {
+    input: object.isRequired,
+    label: string,
+    type: string.isRequired,
+    placeholder: string,
+    meta: object,
+  };
+
+  OptionItem({ value, label }) {
```
Comment (bernabe9) [inline src/components/common/Select.js]: As a convention only the components names start with a capital letter.

This function should be: `optionItem`

File: src/components/user/SignUpForm.js [hunk 2]
```diff
@@ -10,15 +11,20 @@ import {
...
   name: { id: 'login.form.name' },
   email: { id: 'login.form.email' },
   password: { id: 'login.form.password' },
-  passConfirmation: { id: 'signup.form.passconfirmation' }
+  passwordPlaceHolder: { id: 'login.form.password.placeholder' },
+  passConfirmation: { id: 'signup.form.passconfirmation' },
+  gender: { id: 'signup.form.gender' }
 });
 
+const options = [{ value: 'female', label: 'Female' }, {value: 'male', label: 'Male'}, {value:'other', label: 'Other'}];
```
Comment (bernabe9) [inline src/components/user/SignUpForm.js]: Move this to a constant file

File: src/containers/SignUpPage.js [hunk 3]
```diff
@@ -8,8 +8,13 @@ import { signUp } from 'actions/userActions';
 import SignUpForm from 'components/user/SignUpForm';
 import routes from 'constants/routesPaths';
 
-// Styles<
-import './SignUpPage.css';
+// Styles
+import './../styles/signUpPage.scss';
```
Comment (bernabe9) [inline src/containers/SignUpPage.js]: As a convention for the scss files use dashes for the spaces.

For example: `./../styles/sign-up-case.scss`

File: src/containers/SignUpPage.js [hunk 4]
```diff
@@ -8,8 +8,13 @@ import { signUp } from 'actions/userActions';
...
+
+// Imagenes
+import facebookIcon from './..//assets/socialNetworks/facebook@3x.png';
```
Comment (bernabe9) [inline src/containers/SignUpPage.js]: remove extra slash

File: src/styles/signUpPage.scss [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+    font-weight: 600;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: normal;
+    letter-spacing: 1.5px;
+    text-align: center;
+    color: $gray-color;
+}
+
+#signup .slide{
```
Comment (bernabe9) [inline src/styles/signUpPage.scss:25]: Always avoid using ids for selectors

File: src/styles/signUpPage.scss [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+    float: left;
+    width: 50%;
+    height: 1000px;
+}
+
+#signup #slideLeft {
```
Comment (bernabe9) [inline src/styles/signUpPage.scss:31]: same

File: src/styles/signUpPage.scss [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+        font-size: 10px;
+        font-weight: 600;
+        font-style: normal;
+        font-stretch: normal;
+        line-height: normal;
+        letter-spacing: 1.5px;
+        color: $gray-color;
+    }
+
+    #title {
```
Comment (bernabe9) [inline src/styles/signUpPage.scss:106]: same

**Review by bernabe9 — CHANGES_REQUESTED** [review]

File: src/containers/SignUpPage.js (hunk 3 above)
Comment (mdamonte88) [reply src/containers/SignUpPage.js]: I fixed it.

File: src/containers/SignUpPage.js (hunk 4 above)
Comment (mdamonte88) [reply src/containers/SignUpPage.js]: I agree with you.

File: src/components/user/SignUpForm.js (hunk 2 above)
Comment (mdamonte88) [reply src/components/user/SignUpForm.js]: what about the internationalization ?  What should I import?

File: src/containers/SignUpPage.js [hunk 8]
```diff
@@ -25,13 +28,23 @@ class SignUpPage extends PureComponent {
...
 
     return (
-      <div id="signup">
-        <p id="title"><FormattedMessage id="signup.title" /></p>
-        <SignUpForm onSubmit={signUp} />
-        <hr/>
-        <Link className="SIGN-IN-Copy" to={routes.login}>
-          <FormattedMessage id="signup.signin" />
-        </Link>
+      <div id="signup" className="slidesContainer">
```
Comment (bernabe9) [inline src/containers/SignUpPage.js]: remove this id

**Review by bernabe9 — CHANGES_REQUESTED** [review]

**Review by bernabe9 — APPROVED** [review]

**Pull request merged**